packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js

```diff
@@ -120,7 +120,12 @@ class SpecReporter extends WDIOReporter {
     if (spec.indexOf(`${path.sep}packages${path.sep}`) > -1) {
       return spec.split(`${path.sep}packages${path.sep}`).pop().split(path.sep)[0];
     }
-
+    const packageJsonPath = path.join(process.cwd(), 'package.json');
+    if (fs.existsSync(packageJsonPath)) {
+      const jsonString = fs.readFileSync(packageJsonPath);
+      const packagejson = JSON.parse(jsonString);
+      return packagejson.name;
```

Comment:
on second thought, functional-testing already pulls in fs-extra, which has a [`readJsonSync`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md) method that reads and parses json. this requires us to replace our `fs` import with  `fs-extra`, what do you think?

```suggestion
      const { name } = fs.readJsonSync(packageJsonPath);
      return name;
```

Reply:
Yea i think that works. cuts down on the code a little and we already have fs-extra in the package so no reason not to.

Reply:
made the changes here, got a draft pr going to test the changes to make sure i didn't break anything